draft-ietf-core-coap-tcp-tls.md

```diff
@@ -684,19 +684,27 @@ option is understood but cannot be processed, the option documents the behavior.
 
 ## Capability and Settings Messages (CSM)
 
-Capability and Settings messages are used for two purposes:
+Capability and Settings messages (CSM) are used for two purposes:
 
 * Each capability option advertises one capability of the sender to the recipient. 
 
 * Setting options indicate a setting that will be applied by the sender.
 
-Most CSM options are useful mainly as initial messages in the connection.
+Most CSM Options are useful mainly as initial messages in the connection.
 
-Both capability and and setting options are cumulative,
-i.e., a capability message without any option is a no-operation (and
+Both capability and settings options are cumulative,
+i.e., a Capability and Settings message does not invalidate a previously
+made capability indication or setting even if that is not repeated, and
+a capability message without any option is a no-operation (and
 can be used as such). An option that is given might override a
 previous value for the same option; the option defines how to handle
-this, if needed. 
+this, if needed.
+
+Default values given below for CSM Options
+are not default values for the option (this would mean that an
+empty Capability and Settings message would set the option back to its

```

Comment:
I see. You're concerned about the language related to defaults in https://tools.ietf.org/html/rfc7252#section-5.4.4